source/guides/stacks/mers.rst

```diff
@@ -44,23 +60,23 @@ The steps in this guide use |CL-ATTR| as the host system.
 
 .. important:: 
 
-   For optimal performance, a processor with Vector Neural Network
-   Instructions (VNNI) should be used. VNNI is an extension of Intel® 
-   Advanced Vector Extensions 512 (Intel® AVX-512) and is available starting 
-   with the 2nd generation of Intel® Xeon® Scalable Platform, providing AI 
+   For optimal media analytics performance, a processor with Vector Neural
+   Network Instructions (VNNI) should be used. VNNI is an extension of Intel®
+   Advanced Vector Extensions 512 (Intel® AVX-512) and is available starting
+   with the 2nd generation of Intel® Xeon® Scalable Platform, providing AI
```

Comment:
Intel Xeon approved nouns dont include "Scalable Platform". Approved nouns include: Scalable processor OR platform e.g. "Intel® Xeon® Scalable processor" or "Intel® Xeon® platform". Check with me if we need to track down the correct name here.